android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -1179,19 +1180,31 @@ class JsonConverter {
          * Converts the [json] to a `RNUiConfig` object.
          */
         fun toPlayerViewConfig(json: ReadableMap) = PlayerViewConfig(
-            uiConfig = UiConfig.WebUi(
-                playbackSpeedSelectionEnabled = json.getMap("uiConfig")
-                    ?.getBooleanOrNull("playbackSpeedSelectionEnabled")
-                    ?: true,
-            ),
+            uiConfig = if (json.getMap("styleConfig")?.getBooleanOrNull("isUiEnabled") == false) {
+                UiConfig.Disabled
+            } else {
```

Comment:
I think this is already taken care of in the native player (cc @strangesource)

Reply:
I thought so too, but the `isUiEnabled` didn't work for me at all before adding this line :shrug: 

maybe @strangesource can shed some light :thinking:

Reply:
As we provide both, PlayerViewConfig and StyleConfig, PlayerViewConfig overrides the StyleConfig values. See https://cdn.bitmovin.com/player/android/3/docs/player-ui-web/com.bitmovin.player/-player-view/config.html for reference.

Reply:
I'll create a ticket to completely remove the deprecated (internal) `StyleConfig` usage in RN/Android.

Reply:
https://github.com/bitmovin/bitmovin-player-react-native/pull/316/files#r1388972875 still holds. What do you think about this?

Reply:
removed with https://github.com/bitmovin/bitmovin-player-react-native/pull/316/commits/a729aae634392043ada8ca50648c290eb297189f